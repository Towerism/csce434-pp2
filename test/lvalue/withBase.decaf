int asdf() {
    (asdf = 5).qwer;
}