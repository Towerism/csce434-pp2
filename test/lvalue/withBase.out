
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
  2         FieldAccess: 
  2            AssignExpr: 
  2               FieldAccess: 
  2                  Identifier: asdf
  2               Operator: =
  2               IntConstant: 5
  2            Identifier: qwer
